src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
 
 function bindCopyEvents() {
-
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('#copyTableModal > tbody > tr').on('click', function(e){
         e.preventDefault();
-
+        
         var currentRow = $(this);
-        if (currentRow.hasClass('row-selected')) {
-            return;
+        
+        if (typeof prevRow != 'undefined') {
+            $(prevRow).removeClass('row-selected');
```

Comment:
if prevRow was named as $prevRow (better if $previouslySelectedCopyRow), there will be no confusion over whether there is a need to rewrap the prevRow object as a jQuery wrapped object. Note that prevRow is always assigned to currentRow at line 197 and currentRow is always wrapped in a jQuery object before we reach this block.